src/test/groovy/com/intuit/graphql/orchestrator/integration/NestedLevelStitchingSpec.groovy

```diff
@@ -142,5 +177,16 @@ class NestedLevelStitchingSpec extends BaseIntegrationTestSpecification {
         aType.description.contains("description for schema2")
     }
 
+    def "deprecated fields can not be referenced again"() {
+        given:
+        topLevelDeprecatedService1 = createSimpleMockService("test1", conflictingSchema, new HashMap<String, Object>())
+        topLevelDeprecatedService2 = createSimpleMockService("test2", conflictingSchema2, new HashMap<String, Object>())
+
+        when:
+        specUnderTest = createGraphQLOrchestrator([topLevelDeprecatedService1, topLevelDeprecatedService2])
 
+        then:
+        def exception = thrown(StitchingException)
+        exception.message == "[test2,test1] are reusing object type NestedType with different field definitions."
```

Comment:
change to:
FORBIDDEN: Subgraphs [test2,test1] are reusing type NestedType with different field definitions.